src/components/common/Button/index.stories.tsx

```diff
@@ -69,3 +67,17 @@ DisabledButton.args = {
   kind: ButtonVariations.disabled,
   disabled: true,
 }
+
+export const BigButton = Template.bind({})
+BigButton.args = {
+  label: 'Big Button',
+  kind: ButtonVariations.primary,
+  size: 'big',
```

Comment:
enum in one but not the other?

Reply:
i was waiting for this comment :sweat_smile: I am not sure which to go with. I'm leaning towards your previous comment of using the `type` instead (for both)

Reply:
opted for `type`